en/hardware/reference_design.md

```diff
@@ -30,18 +30,13 @@ The diagram below shows the division of bus and functional responsibilities betw
 
 Some Pixhawk-series controllers are built without the I/O board in order to reduce space or complexity, or to better address certain board use-cases.
 
-The I/O board is disabled by setting parameter [SYS_USE_IO=0](../advanced_config/parameter_reference.md#SYS_USE_IO).
-When the I/O board is disabled:
-- The MAIN mixer file is loaded into the FMU (so the "MAIN" outputs listed in the [Airframe Reference](../airframes/airframe_reference.md) appear on the port labeled AUX).
-  The AUX mixer file isn't loaded, so outputs defined in this file are not used.
-- RC input goes directly to the FMU rather than via the IO board.
-
-Flight controllers without an I/O board have `MAIN` ports, but they *do not* have `AUX` ports.
-Consequently they can only be used in [airframes](../airframes/airframe_reference.md) that do not use `AUX` ports, or that only use them for non-essential purposes (e.g. RC passthrough).
-They can be used for most multicopters and *fully* autonomous vehicles (without a safety pilot using RC control), as these typically only use `MAIN` ports for motors/essential controls.
+In this case [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) is set to `0` so that the I/O driver is not started.
+Because there are no I/O board outputs, the [Actuator Configuration](../config/actuators.md) will not display the MAIN tab.
+Airframe actuator configurations should typically assign essential outputs to the AUX outputs (FMU) so that the configuration can still be used on controllers that do not have MAIN outputs (I/O board).
```

Comment:
Is this actually relevant?
Either way it's the opposite (yes it's confusing): it's the MAIN params/outputs that are used on both.

Reply:
I know why I wrote this - I thought with control allocation FMU was always fixed to being AUX in all cases. This may clear up depending on your answer ^^^^

Historically advice was to use MAIN (IO) for essential outputs (i.e. motor 1 to X) and AUX (FMU) was used for less essential (gripper servo say). On board without IO MAIN was mapped to FMU so this advice still held - MAIN was always present.

From what I can tell with control allocation, MAIN will be present if you have an IO board or for targets that only have an FMU (so they map it to MAIN). MAIN will not be present on cases like the PX4 mini which only have AUX. Upshot, the advice is wrong in one of those two cases. 

> it's the MAIN params/outputs that are used on both.

The actuator docs says "PWM AUX outputs are preferred over the PWM MAIN outputs for controlling motors (they have lower latency)."; 

What should I say? :-)

Reply:
Correct.

> "PWM AUX outputs are preferred over the PWM MAIN outputs for controlling motors (they have lower latency)."

This is also still correct. You could add "If the board has an I/O chip, PWM AUX outputs are preferred..."

Reply:
I think the current way is fine/more useful - or at least adding the detailed explanation won't help anyone.

I've removed the suggestion that main should always be used since you can no longer reliably assign to main and know the configuration will work on all airframes.